src/test/test-dirent-util.c

```diff
@@ -26,7 +26,8 @@ TEST (test_dirent_ensure_type) {
         assert_se(de.d_type == DT_UNKNOWN);
```

Comment:
I don't understand this line BTW. There's no need to assert on something we just defined?